projects/models.py

```diff
@@ -365,3 +365,15 @@ def __repr__(self) -> str:
             for ck in self.SOURCE_REPORT_FIELDS
         ]
         return "\n" + "\n".join(lines)
+
+
+class PublicationDuplicate(models.Model):
```

Comment:
I think we could simplify this without a new model. We could just mark `Publication`s with status `'duplicate'` and then add a nullable field `duplicates`/`duplicate_of` which points to another publication.

Reply:
We can simply by adding a duplicate_of nullable field to the Publication model but the reason for using another model is to isolate the Publication model/table and not make many changes to it